src/main/java/com/playmonumenta/structures/commands/ActivateSpecialStructure.java

```diff
@@ -17,20 +20,15 @@ public static void register(Plugin plugin) {
 		final String command = "activatespecialstructure";
 		final CommandPermission perms = CommandPermission.fromString("monumenta.structures");
 
-		new CommandAPICommand(command)
-			.withPermission(perms)
-			.withArguments(new StringArgument("label").replaceSuggestions(RespawnManager.SUGGESTIONS_STRUCTURES))
-			.executes((sender, args) -> {
-				activate(sender, plugin, (String)args[0], null);
-			})
-			.register();
+		Argument<String> labelArg = new StringArgument("label").replaceSuggestions(RespawnManager.SUGGESTIONS_STRUCTURES);
+		TextArgument pathArg = new TextArgument("special_structure_path");
 
 		new CommandAPICommand(command)
 			.withPermission(perms)
-			.withArguments(new StringArgument("label").replaceSuggestions(RespawnManager.SUGGESTIONS_STRUCTURES))
-			.withArguments(new TextArgument("special_structure_path"))
+			.withArguments(labelArg)
+			.withOptionalArguments(pathArg)
```

Comment:
Should this be optional? it's nullable in there, but in implementation it's checking if null or empty and forcing failure

Reply:
Yeah, looks like it. Not sure why it was written before to have a command that always failed... Fixing